README.md

```diff
@@ -122,4 +122,13 @@ It sends an event to the uinput device.
 * *code* `Number`
 * *callback* `Function`
 
-Wrapper over send_event to simulate key presses and mouse clicks.
\ No newline at end of file
+Wrapper over send_event to simulate key presses and mouse clicks.
+
+### emit_combo(stream, code, callback)
+
+* *stream* `WritableStream`
+* *code* `Array with shift or alt and number or letter`
+* *callback* `Function`
```

Comment:
The description of `code` is not very clear to me. After reading the code it looks like an array with any combination of keys, right?